### PR TITLE
Onav 1016 

### PR DESCRIPTION
**Pull request by dobrienCP**
Contains the changes to the documentation reflecting the new Mission Manager UI updates.

### CODE REVIEW

File: docs_outdoornav_user_manual/faq.mdx [hunk 1]
```diff
@@ -29,57 +29,36 @@ toc_max_heading_level: 4
...
-    Check the Goals that are enabled in your Goal Panel. If you see a
-    red warning triangle beside the **Save Image** task it means that
-    the settings for this task were not set. Set them and rerun the
+    Check the Waypoints that have a **Save Image** task in the Waypoint panel. 
+    If you see a red warning triangle beside the **Save Image** task it means 
+    that the settings for this task were not set. Set them and rerun the
     mission.
 
-8.  **The navigation gets stuck on the ""Compute Path"" feedback.
+6.  **The navigation gets stuck on the ""Compute Path"" feedback.
```
Comment (jhiggins-cpr) [inline docs_outdoornav_user_manual/faq.mdx]: Can we change from "" to "?

File: docs_outdoornav_user_manual/web_user_interface/ui_autonomous_mode.mdx [hunk 2]
```diff
@@ -26,23 +26,18 @@ software architecture section for more information.
...
-    not the last point. The list of Viapoints forms the path for the
-    specific Goal.
+    position relative to the datum in meters. 
 -   **Task** A Task is an automated activity or wait time implemented as
-    a ROS action. Tasks are called in the order they are added to a
-    Goal.
+    a ROS action at a specific Waypoint. Tasks are called in the order they are 
+    added to a Waypoint.
+-   **Ghost Waypoint** A transparent waypoint that is not part of the mission. This
+    Waypoint appears inbetween two other waypoints when in edit mode. The user can
```
Comment (jhiggins-cpr) [inline docs_outdoornav_user_manual/web_user_interface/ui_autonomous_mode.mdx]: "inbetween" -> "between"

File: docs_outdoornav_user_manual/web_user_interface/ui_autonomous_mode.mdx [hunk 3]
```diff
@@ -26,23 +26,18 @@ software architecture section for more information.
...
-    specific Goal.
+    position relative to the datum in meters. 
 -   **Task** A Task is an automated activity or wait time implemented as
-    a ROS action. Tasks are called in the order they are added to a
-    Goal.
+    a ROS action at a specific Waypoint. Tasks are called in the order they are 
+    added to a Waypoint.
+-   **Ghost Waypoint** A transparent waypoint that is not part of the mission. This
+    Waypoint appears inbetween two other waypoints when in edit mode. The user can 
+    drag and drop this ghost waypoint to add a new waypoint to the mission in between
```
Comment (jhiggins-cpr) [inline docs_outdoornav_user_manual/web_user_interface/ui_autonomous_mode.mdx]: "in between" -> "between"

File: docs_outdoornav_user_manual/web_user_interface/ui_autonomous_mode.mdx [hunk 4]
```diff
@@ -64,94 +59,24 @@ To access the Map Settings: Menu → Settings → Map:
...
-1.  Defining a point on the map as a Goal Point.
-2.  Adding Task for the Goal.
-3.  Adding Viapoints for the Goal to help guide the UGV from its
-    starting point to the Goal.
-4.  Repeating the steps above to add more Goals (if needed).
-5.  Specifying the order of the Goals.
-6.  Saving the Mission.
-
-<iframe width="680" height="350" src="https://www.youtube.com/embed/zcD5shs5wZ4" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+To create a new Mission to run on the UGV first ensure that the UI is in "Edit Mode" (
```
Comment (jhiggins-cpr) [inline docs_outdoornav_user_manual/web_user_interface/ui_autonomous_mode.mdx]: "To create a new Mission to run on the UGV" -> "To create a new Mission"

File: docs_outdoornav_user_manual/web_user_interface/ui_autonomous_mode.mdx [hunk 5]
```diff
@@ -64,94 +59,24 @@ To access the Map Settings: Menu → Settings → Map:
...
+select the pencil icon in the bottom bar). Then open the drop down menu in the bottom
+bar and select the "Add Mission" option. This will allow you to create a new mission 
+which you can then add Waypoints to.
```
Comment (jhiggins-cpr) [inline docs_outdoornav_user_manual/web_user_interface/ui_autonomous_mode.mdx]: "to which you can then add Waypoints."

File: docs_outdoornav_user_manual/web_user_interface/ui_autonomous_mode.mdx [hunk 6]
```diff
@@ -64,94 +59,24 @@ To access the Map Settings: Menu → Settings → Map:
...
-  </figure>
-</center>
-
-<center>
-  <figure>
-    <img src="/img/outdoornav_images/ui_route_end.png" />
-    <figcaption>Route Mode: Step 6</figcaption>
-  </figure>
-</center>
+As waypoints are placed, a "ghost waypoint" will appear between each real
```
Comment (jhiggins-cpr) [inline docs_outdoornav_user_manual/web_user_interface/ui_autonomous_mode.mdx]: "each real waypoints" -> "each pair of real Waypoints"

File: docs_outdoornav_user_manual/web_user_interface/ui_autonomous_mode.mdx [hunk 7]
```diff
@@ -64,94 +59,24 @@ To access the Map Settings: Menu → Settings → Map:
...
+waypoint and can be dragged to a new spot to insert a real Waypoint in
```
Comment (jhiggins-cpr) [inline docs_outdoornav_user_manual/web_user_interface/ui_autonomous_mode.mdx]: "in between" -> "between"

File: docs_outdoornav_user_manual/web_user_interface/ui_autonomous_mode.mdx [hunk 8]
```diff
@@ -167,65 +92,60 @@ over the planned route as can be seen in the following figure.
...
 
-To rename a Goal follow these steps:
+To rename a Waypoint follow these steps:
 
-1.  Enable the "Goal Panel" toggle. If you cannot find this button,
-    see [Goal Panel](#goal-panel) for reference.
-2.  Click the name of the Goal which the user wants to rename.
+1.  Enable the "Waypoint Panel" toggle. If you cannot find this button,
+    see [Waypoint Panel](#waypoint-panel) for reference.
+2.  Click the name of the Waypoint which the user wants to rename.
```
Comment (jhiggins-cpr) [inline docs_outdoornav_user_manual/web_user_interface/ui_autonomous_mode.mdx]: We flip between the second person singlular ("you") and third person singular ("the user"). We should choose one or the other and be consistent throughout. Applies to other locations in this file as well.

File: docs_outdoornav_user_manual/web_user_interface/ui_autonomous_mode.mdx [hunk 9]
```diff
@@ -257,15 +177,15 @@ To add a Task to the end of a Mission:
     -   **Undock UGV:** <img src="/img/outdoornav_images/undockTask_40px.png" width="30px"/>
         Will undock the UGV from the autocharge dock. Once
         completed, the UGV can be sent on autonomous missions. It is
-        often recommended to place the undock Goal first in the list;
+        often recommended to place the undock task first in the list of Waypoints;
```
Comment (jhiggins-cpr) [inline docs_outdoornav_user_manual/web_user_interface/ui_autonomous_mode.mdx]: "Task"

File: docs_outdoornav_user_manual/web_user_interface/ui_autonomous_mode.mdx [hunk 10]
```diff
@@ -257,15 +177,15 @@ To add a Task to the end of a Mission:
     -   **Undock UGV:** <img src="/img/outdoornav_images/undockTask_40px.png" width="30px"/>
         Will undock the UGV from the autocharge dock. Once
         completed, the UGV can be sent on autonomous missions. It is
-        often recommended to place the undock Goal first in the list;
+        often recommended to place the undock task first in the list of Waypoints;
         that way, the UGV will automatically continue towards its next
-        Goal once undocked. See Section
+        waypoint once undocked. See Section
```
Comment (jhiggins-cpr) [inline docs_outdoornav_user_manual/web_user_interface/ui_autonomous_mode.mdx]: "Waypoint"

File: docs_outdoornav_user_manual/web_user_interface/ui_autonomous_mode.mdx [hunk 11]
```diff
@@ -283,66 +203,56 @@ To add a Task to the end of a Mission:
 
 <iframe width="696" height="358" src="https://youtube.com/embed/V6K2VXM5EaU" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
-#### Goal Heading
+#### Waypoint Heading
 
-When creating a Goal, the user has the option of setting a final heading
-for the Goal. For example, when creating a Goal to an inspection point,
+When creating a Waypoint, the user has the option of setting a final heading
+for the Waypoint. For example, when creating a mission to an inspection point,
```
Comment (jhiggins-cpr) [inline docs_outdoornav_user_manual/web_user_interface/ui_autonomous_mode.mdx]: "mission" -> "Waypoint"

File: docs_outdoornav_user_manual/web_user_interface/ui_autonomous_mode.mdx [hunk 12]
```diff
@@ -283,66 +203,56 @@ To add a Task to the end of a Mission:
...
 </center>
 
-The heading that has been entered will only be applied to the Goal Point
-(ie. the robot will only align itself with the correct heading at the
-end of the goal). If you need the robot to be at specific headings at
-some of the Goal's Viapoints, the only way to do this is to split the
-longer Goal into smaller goals and apply your required headings as
-required.
+The heading that has been entered will only be applied to the Waypoint
+(ie. the robot will only align itself with the correct heading at The
```
Comment (jhiggins-cpr) [inline docs_outdoornav_user_manual/web_user_interface/ui_autonomous_mode.mdx]: "The waypoint" -> "the Waypoint"

File: docs_outdoornav_user_manual/web_user_interface/ui_autonomous_mode.mdx [hunk 13]
```diff
@@ -283,66 +203,56 @@ To add a Task to the end of a Mission:
...
+waypoint). If you need the robot to be at specific headings at
```
Comment (jhiggins-cpr) [inline docs_outdoornav_user_manual/web_user_interface/ui_autonomous_mode.mdx]: "Waypoint"

File: docs_outdoornav_user_manual/web_user_interface/ui_autonomous_mode.mdx [hunk 14]
```diff
@@ -283,66 +203,56 @@ To add a Task to the end of a Mission:
...
+other waypoints you'll need to enter these in for each specific Waypoint.
```
Comment (jhiggins-cpr) [inline docs_outdoornav_user_manual/web_user_interface/ui_autonomous_mode.mdx]: "Waypoints"

File: docs_outdoornav_user_manual/web_user_interface/ui_autonomous_mode.mdx [hunk 15]
```diff
@@ -283,66 +203,56 @@ To add a Task to the end of a Mission:
...
-
 ## Mission Execution
 
 ### Start Mission
 
 At the bottom of the UI, the user has the ability to start the currently
 selected mission by clicking the "Play" button <img src="/img/outdoornav_images/ui_start_button.png" width="16px"/>. When the
-mission has been started this button will turn green. Starting a mission
-consists of sending the currently enabled set of goals.
+mission has been started this button will turn green.
```
Comment (jhiggins-cpr) [inline docs_outdoornav_user_manual/web_user_interface/ui_autonomous_mode.mdx]: "Mission"

File: docs_outdoornav_user_manual/web_user_interface/ui_overview.mdx [hunk 16]
```diff
@@ -40,12 +40,18 @@ mode.
 6.  **GPS Status Indicator:** The GPS status indicators will display GPS
     signal accuracy for position (POS indicator) and heading (DIR
     indicator). Green indicators represent RTK accuracy and are
-    currently required for accurate autonomous navigation. Noticeable
-    oscillations may occur if the indicators are yellow. Red indicators
-    mean no GPS signal and autonomous navigation missions should not be
-    started.
+    currently required for accurate autonomous navigation. Yellow and orange 
+    indicators represent SBAS and SPP accuraciy respectively and noticeable
```
Comment (jhiggins-cpr) [inline docs_outdoornav_user_manual/web_user_interface/ui_overview.mdx]: "accuracy"

File: docs_outdoornav_user_manual/web_user_interface/ui_overview.mdx [hunk 17]
```diff
@@ -55,25 +61,23 @@ mode.
...
-11. **Save Mission Button:** Save the current mission. This will store
-    the the current mission to the database.
-
 12. **Edit Mission Toggle:** This toggle allows the user to start
-    creating Goals for the current Mission. Once this button is enabled,
-    the Waypoint Mode and Route Mode buttons will now be available for
-    selection. When a Mission is started this toggle will be disabled
-    (ie. the Operator will not be able to modify/add Goals).
+    creating Waypoints for the current Mission. The user will also be able to
+    drag and drop waypoints in this mode. Once this button is enabled,
```
Comment (jhiggins-cpr) [inline docs_outdoornav_user_manual/web_user_interface/ui_overview.mdx]: "Waypoints"

File: docs_outdoornav_user_manual/web_user_interface/ui_overview.mdx [hunk 18]
```diff
@@ -82,24 +86,19 @@ mode.
...
-    again will start a new Goal.
+14. **Waypoint Drop Button:** The Waypoint indicator marker on the
+    bottom bar allows the user to place a Waypoint at the location of the UGV. 
 
-15. **Start Button:** Start the current mission (ie. execute the list of
-    enabled Goals sequentially).
+15. **Start Button:** Start the current mission.
 
-16. **Pause Button:** Pause the current Goal. Pressing the start button
+16. **Pause Button:** Pause the current mission. Pressing the start button
```
Comment (jhiggins-cpr) [inline docs_outdoornav_user_manual/web_user_interface/ui_overview.mdx]: "Mission"

**Review by jhiggins-cpr — CHANGES_REQUESTED** [review]
Overall, looks ok to me, once we complete the minor changes noted throughout.

File: docs_outdoornav_user_manual/web_user_interface/ui_autonomous_mode.mdx (hunk 8 above)
Comment (dobrienCP) [reply docs_outdoornav_user_manual/web_user_interface/ui_autonomous_mode.mdx]: I'll try and keep it in 3rd person for now.

**Review by jhiggins-cpr — APPROVED** [review]

**Pull request merged**